consul/aio.py

```diff
@@ -42,8 +43,10 @@ def get(self, callback, path, params=None):
         uri = self.uri(path, params)
         return self._request(callback, 'GET', uri)
 
-    def put(self, callback, path, params=None, data=''):
+    def put(self, callback, path, params=None, data='', json=None):
```

Comment:
Do we need a new parameter here? Probably best to stick to current pattern (eg: https://github.com/cablehead/python-consul/blob/master/consul/base.py#L847) I have not looked at this in detail, but have a feeling this might not be required and will avoid a few of the other changes. Same for the other similar changes.